TASK01.md

```diff
@@ -2,7 +2,7 @@
 
 1. Faça um fork deste projeto dentro da sua conta no GitHub
 2. Clone o projeto "forked" em seu ambiente de desenvolvimento favorito
-3. Crie as classes e códigos necessários para:
+3. Crie as classes e códigos necessários para:  

```

Comment:
How Markdown don't accept numbered sublists you have to force a new line and simulate this (`3.1` is part of the sublist body, not the list itself), to do this, just put 2 spaces after the last line.